conformance/lang/expressions/multiplicative-expr/division.balt

```diff
@@ -0,0 +1,4506 @@
+
+     a = -17.E12d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output -Infinity
+
+     a = 11.E12d;
+     b = -0.0d;
+     c = a / b;
+     io:println(c); // @output Infinity
+
+     a = 17.E1290d;
+     b = 13.E1521d;
+     c = a / b;
+     io:println(c); // @output 1.307692307692307692307692307692308E-231
+
+     a = 0.0d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output NaN
+
+     a = -0.0d;
+     b = 0.0d;
+     c = a / b;
+     io:println(c); // @output NaN
+
+     a = 17.E1290d;
+     b = ();
+     c = a / b;
+     io:println(c.toBalString()); // @output ()
+
+     a = ();
+     b = 13.E1521d;
+     c = a / b;
+     io:println(c.toBalString()); // @output ()
+}
+
+Test-Case: output
+Description: Test division scenarios for different float values.
+Labels: multiplicative-expr, numeric-literal, float-point-literal
+
+public function main() {
+     float a = 2.5E-17;
+     float b = 2.5E+17;
+     float c = b / a;
+     io:println(c); // @output 1.0000000000000001E34
+     io:println(2.5E+17 / 2.5E-17); // @output 1.0000000000000001E34
+
+     a = 1.423223E6;
+     b = 2.34413E2;
+     c = a / b;
+     io:println(c); // @output 6071.433751541083
+     io:println(1.423223E6 / 2.34413E2); // @output 6071.433751541083
+
+     a = 25.E1742f;
+     b = 1.0f;
+     c = a / b;
+     io:println(c); // @output Infinity
+     io:println(25.E1742f / 1.0f); // @output Infinity
+
+     a = 25.E1742f;
+     b = 0.0f;
+     c = a / b;
+     io:println(c); // @output Infinity
+     io:println(25.E1742f / 0.0f); // @output Infinity
+
+     a = 25.E17f;
+     b = 0.9E-10f;
+     c = a / b;
+     io:println(c); // @output 2.7777777777777776E28
+     io:println(25.E17f / 0.9E-10); // @output 2.7777777777777776E28
+
+     a = -25.E17f;
+     b = 0.9E-10f;
+     c = a / b;
+     io:println(c); // @output -2.7777777777777776E28
+     io:println(-25.E17f / 0.9E-10); // @output -2.7777777777777776E28
+
+     a = 17.E12f;
+     b = 0.0f;
+     c = a / b;
+     io:println(c); // @output Infinity
+     io:println(17.E12f / 0.0f); // @output Infinity
+
+     a = -17.E12f;
+     b = 0.0f;
+     c = a / b;
+     io:println(c); // @output -Infinity
+     io:println(-17.E12f / 0.0f); // @output -Infinity
+
+     a = 11.E12f;
+     b = -0.0f;
+     c = a / b;
+     io:println(c); // @output -Infinity
+     io:println(11.E12f / -0.0f); // @output -Infinity
+
+     a = 17.E1290f;
+     b = 13.E1521f;
+     c = a / b;
+     io:println(c); // @output NaN
+     io:println(17.E1290f / 13.E1521f); // @output NaN
+
+     a = 0.0f;
+     b = 0.0f;
+     c = a / b;
+     io:println(c); // @output NaN
+     io:println(0.0f / 0.0f); // @output NaN
+
+     a = -0.0f;
+     b = 0.0f;
+     c = a / b;
+     io:println(c); // @output NaN
+     io:println(-0.0f / 0.0f); // @output NaN
+}
+
+Test-Case: output
+Description: Test division scenarios for different nillable float values.
+Labels: multiplicative-expr, numeric-literal, float-point-literal
```

Comment:
Shall we use `DecimalFloatingPointNumber` instead of `floating-point-literal`?
Don't need `numeric-literal` label?
Missing `float`, `FloatTypeSuffix`, `optional-type`, `numeric-nil-lifting`, `unary-minus`, `nil-literal`, `value:toBalString` labels.